src/test/java/com/salesforce/dynamodbv2/testsupport/ArgumentBuilder.java

```diff
@@ -111,18 +114,7 @@ public TestArgumentSupplier(AmazonDynamoDB rootAmazonDynamoDb) {
          * byAccount
```

Comment:
re line 104 above ("return ImmutableList.of(S, N, B);"): consider "return Arrays.asList(ScalarAttributeType.values());"